src/components/InputField.vue

```diff
@@ -44,63 +45,80 @@ watch(
   { deep: true }
 );
 
+function isLeftStartPoint(): boolean {
+  let leftStartPoint = false;
+  if (
+    crossword.value[crosswordTile.value.positionY][
+      crosswordTile.value.positionX - 1
+    ].startPoint
```

Comment:
```suggestion
    getCrosswordElement(crosswordTile.value.positionX - 1, crosswordTile.value.positionY).startPoint
```

Reply:
getCrosswordElement and crossword.value[][] are different elements. getCrosswordElement gets the specific Tile on the dom and crossword.value[][] gets the object in the crossword array.